src/pages/MemoriesPage.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import MemoriesPage from './MemoriesPage'
+
+describe('MemoriesPage', () => {
+  it('renders two listItems', () => {
+    render(<MemoriesPage />)
+
+    const memoryItems = screen.getAllByRole('listitem')
+    expect(memoryItems).toHaveLength(2)
+  })
```

Comment:
This works only due to your current setting of MemoriesPage, since you don't provide only two listItems in this text but rather have them in your component. I mean it is working and that's nice but you will probably get more out of the test if you make the component listen to list items passed in as props. If that doesn't make sense, think about the test itself, wether the test is necessary. Most of the times a whole page which doesn't receive props is not tested but rather the components inside of it